web/store/issue/module/issue.store.ts

```diff
@@ -301,27 +300,39 @@ export class ModuleIssues extends IssueHelperStore implements IModuleIssues {
     fetchAddedIssues = true
   ) => {
     try {
-      await this.moduleService.addIssuesToModule(workspaceSlug, projectId, moduleId, {
-        issues: issueIds,
-      });
-
-      if (fetchAddedIssues) await this.rootIssueStore.issues.getIssues(workspaceSlug, projectId, issueIds);
-
+      // add the new issue ids to the module issues map
```

Comment:
I feel like addIssuesToModule and removeIssuesFromModule doesn't need optimistic updates. Since it is done with an action. I thing we can await for the API call. We'll only need optimistic updates for issue updates

Reply:
`removeIssuesFromModule` needs to be optimistic as it is done using quick actions and there is no loading state. Also, I do not see any harm in making `addIssuesToModule` optimistic.